ChangeLog.md

```diff
@@ -149,6 +149,8 @@ Other enhancements:
   [#3534](https://github.com/commercialhaskell/stack/issues/3534)
 * GHC bindists can now be identified by their SHA256 checksum in addition to
   their SHA1 checksum, allowing for more security in download.
+* Added new `--tar-files` option to `stack sdist`, that allows to copy
```

Comment:
Since this isn't going to make it into the 1.6 release, can you move this to "unreleased changes"?

Reply:
As far as I understood, it was already in section "Unreleased changes", in subsection "other enhancements" and following versions 1.5.1 are next. Or is it changelog for 1.6 in the top of the file? It's not marked anywhere.

Reply:
It should go in this area https://github.com/Asalle/stack/blob/630a9d63f099970a3db3bf3667a0bba0457328f0/ChangeLog.md#unreleased-changes